static/src/javascripts/boot.js

```diff
@@ -39,14 +38,7 @@ define([
     var domReadyPromise = new Promise(function (resolve) { domReady(resolve); });
 
     var bootStandard = function () {
-        return domReadyPromise.then(function () {
-            return promiseRequire(['bootstraps/standard/main'])
-                .then(function (boot) { boot(); });
-        });
-    };
-
-    var bootEnhanced = function () {
-        return promiseRequire(['bootstraps/enhanced/main'])
+        return promiseRequire(['bootstraps/standard/main'])
             .then(function (boot) { boot(); });
```

Comment:
Yeah, still a few of those around in this file. We may un-promise this library soon